backend/src/apiserver/server/pipeline_upload_server.go

```diff
@@ -98,14 +98,14 @@ func (s *PipelineUploadServer) uploadPipeline(api_version string, w http.Respons
 	glog.Infof("Upload pipeline called")
 	file, header, err := r.FormFile(FormFileKey)
 	if err != nil {
-		s.writeErrorToResponse(w, http.StatusBadRequest, util.Wrap(err, "Failed to read pipeline from file"))
+		s.writeErrorToResponse(w, http.StatusBadRequest, errors.New("Failed to read a pipeline"))
```

Comment:
I think the wrapped error is helpful. What's the rationale behind scrapping it? Maybe it can log to the backend but not be exposed in the frontend.

Reply:
The above applies to all of the suppressed error logs.